### PR TITLE
Merge gear-program into gear (PLEASE DON'T SQUASH)

### PR DESCRIPTION
**Pull request by NikVolf**

### CODE REVIEW

File: program/res/messager/.binpath [hunk 1]
```diff
@@ -0,0 +1 @@
+../../target/wasm32-unknown-unknown/debug/messager
```
Comment (shamilsan) [inline program/res/messager/.binpath:1]: No need to push

Comment (NikVolf) [reply program/res/messager/.binpath]: fixed (force-pushed to not blow-up history)

File: program/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+[dependencies]
+anyhow = "1.0.60"
+blake2-rfc = "0.2.18"
+base64 = "0.13.0"
+color-eyre = { version = "0.6.2", optional = true }
+dirs = "4.0.0"
+env_logger = "0.9.0"
+futures-util = "0.3.21"
+gear-core = { git = "https://github.com/gear-tech/gear.git" }
```
[the remark this answers is not in the document]
Comment (shamilsan) [reply program/Cargo.toml:27]: Need to fix after the Merge indeed.

[the remark this answers is not in the document]
Comment (NikVolf) [reply program/Cargo.toml:27]: I will fix here no worries

File: scripts/src/test.sh [hunk 3]
```diff
@@ -30,7 +30,7 @@ EOF
 }
 
 workspace_test() {
-  cargo nextest run --workspace "$@" --profile ci
+  cargo nextest run --workspace "$@" --profile ci --exclude gear-program
```
Comment (NikVolf) [inline scripts/src/test.sh:33]: excluded tests for `gear-program` for a while -- it should use node from actual repo

**Review by SabaunT — APPROVED** [review]

**Pull request merged**